<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratePasswordClass</class>
 <widget class="QMainWindow" name="GeneratePasswordClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneratePassword</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="gen_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="text">
     <string>Generuj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ilosc różnych znaków:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generator Hasła</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_length">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dlugosc znakow:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distinct">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>P$M</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_result">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oto twoje hasło:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>tu cos jeszcze bedczied</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distinct_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bachk_button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
     </font>
    </property>
    <property name="text">
     <string>Powrót</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>gen_button</sender>
   <signal>clicked()</signal>
   <receiver>GeneratePasswordClass</receiver>
   <slot>generate_passwor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bachk_button</sender>
   <signal>clicked()</signal>
   <receiver>GeneratePasswordClass</receiver>
   <slot>back_to_signu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>805</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>generate_passwor()</slot>
  <slot>back_to_signu()</slot>
 </slots>
</ui>
